tools/epoch-sync/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "epoch-sync-tool"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+repository.workspace = true
+license.workspace = true
+publish = false
+
+[dependencies]
+anyhow.workspace = true
+clap.workspace = true
+tracing.workspace = true
+
+nearcore.workspace = true
+near-chain.workspace = true
+near-chain-configs.workspace = true
+near-epoch-manager.workspace = true
+near-primitives.workspace = true
+near-store.workspace = true
+
+[features]
+new_epoch_sync = ["nearcore/new_epoch_sync"]
```

Comment:
Since this crate currently only does anything at all when this feature is enabled, wouldn’t it be better if the dependency on the crate itself was optional over implementing a crate feature like this right now?

Reply:
I tried my best :) https://github.com/near/nearcore/pull/10146/commits/67aea72d534a4b3ded27f5c0f10c4adc0979e1c3
Please, check this commit